sos/policies/debian.py

```diff
@@ -14,7 +14,7 @@ class DebianPolicy(LinuxPolicy):
     _debv_filter = ""
     valid_subclasses = [DebianPlugin]
     PATH = "/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games" \
-           + ":/usr/local/sbin:/usr/local/bin"
+        + ":/usr/local/sbin:/usr/local/bin:/snap/bin"
```

Comment:
Nack. You don't just hide a significant change to a long-standing distro policy by tucking it away at the end of pages of Juju plugin patches. This needs to be a separate commit where it can easily be found, and where it can be discussed with the Debian and Ubuntu policy maintainers.